### PR TITLE
Update some documents of minio-go

### PR DESCRIPTION
**Pull request by Diracal**
Update some documents, mainly the Chinese documents,  under the directory of docs according to the last documents .

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -1541,22 +1542,22 @@ if err != nil {
 ### GetBucketLifecycle(bucketName) (lifecycle string, error)
 Get lifecycle on a bucket or a prefix.
 
-__Parameters__
+__参数__
```
Comment (kannappanr) [inline docs/API.md]: These should be removed

File: docs/API.md [hunk 2]
```diff
@@ -1541,22 +1542,22 @@ if err != nil {
...
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
 |`bucketName`  | _string_  |Name of the bucket   |
 
-__Return Values__
+__返回类型__
```
Comment (kannappanr) [inline docs/API.md]: These should be removed. This is the english version of the document.

File: docs/API.md [hunk 3]
```diff
@@ -1541,22 +1542,22 @@ if err != nil {
...
 
 
 |Param   |Type   |Description   |
 |:---|:---| :---|
 |`lifecycle`  | _string_ |Lifecycle returned from the server |
 |`err` | _error_  |Standard Error  |
 
-__Example__
+__示例__
```
Comment (kannappanr) [inline docs/API.md]: These should be removed. This is the english version of the document.

File: docs/API.md (hunk 2 above)
Comment (Diracal) [reply docs/API.md]: Thank you very much, I'll correct it as soon as I can.

File: docs/API.md (hunk 1 above)
Comment (Diracal) [reply docs/API.md]: Thank you very much, I'll correct it as soon as possible.

File: docs/API.md (hunk 3 above)
Comment (Diracal) [reply docs/API.md]: Thank you very much, I'll modify it.

**Comment by harshavardhana** [comment]
Closing this PR as stale - please re-open when it's ready for review.

**Pull request closed without merging**